<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Giriş Ekranı</string>
  </property>
  <widget class="QLabel" name="usernameLbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Kullanıcı Adı</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Parola</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="usernameTxt">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>271</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="passwordTxt">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>271</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Giriş</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
